### DISCUSSION TITLE
Papers background image gets scaled as we zoom-in/out

### DISCUSSION THREAD

**Discussion by DarkMagus** [clientIO/joint#2641] [Q&A]
### Introduction

When an image is used as background of a paper, every time a zoom event occurs the background image gets zoomed too. Is there a way to stop the background image from scaling?
I realize also the same zooming happens when using drawGrid: { name: 'mesh'}. The mesh lines expand/retract as we zoom in /out.

### Steps to reproduce

Below is the code I use to construct the background image of a paper. The image just creates a background similar to JointJS mesh.
```
paper = new joint.dia.Paper({
            model: graph,
            defaultConnectionPoint: {name: 'boundary', args: {offset: 5}},
            defaultAnchor: { name: 'modelCenter' },
            defaultConnector: { name: 'normal' },
            async: true,
            frozen: true,
            interactive: {
                linkMove: false,
                labelMove: false,
                elementMove: false
            },
            sorting: joint.dia.Paper.sorting.APPROX,
            cellViewNamespace: {...allshapes},
            background: {
                color: '#F8F9FA',
                image: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAIElEQVQoU2NkIALcuvU/jZEIdQyjCvGGEjh4QAQxYQkAIfcfBcQyCEYAAAAASUVORK5CYII=',
                repeat: 'repeat',
            }
        });
```

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You need to specify the `size` of the background image.

> size is an object { width: Number, height: Number } defining the background image size. It also allows to use all the CSS background-size property values. In that case all the paper transformations have no impact on the background size. It defaults to auto auto.

```ts
background: {
    color: '#F8F9FA',
    image: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAIElEQVQoU2NkIALcuvU/jZEIdQyjCvGGEjh4QAQxYQkAIfcfBcQyCEYAAAAASUVORK5CYII=',
    repeat: 'repeat',
    size: '20px 20px'
}
```

**Reply by DarkMagus** [reply]
@kumilingus 
it works, 
Thanks lot
